assets/panel/debugger.properties

```diff
@@ -510,6 +510,11 @@ editor.conditionalPanel.close=Cancel edit breakpoint and close
 editor.jumpToMappedLocation1=Jump to %S location
 editor.jumpToMappedLocation1.accesskey=m
 
+# LOCALIZATION NOTE (downloadFile.label): Context menu item
+# for downloading a source's content
+downloadFile.label=Download file
+downloadFile.accesskey=d
```

Comment:
Comment for posterity: access keys are case sensitive. By using lowercase, the last character will be underlined, instead of the first one. Not sure if it's the desired behavior.

If there's no perfect match (same letter and case), it would fall back to the other case, so (`Down` and `d` would underline the first character as fall back).